### ISSUE TITLE
False positive in 'Image not in the accessibility tree is decorative'

### ISSUE THREAD

**Issue by dd8** [auto-wcag/auto-wcag#2335]
https://www.w3.org/WAI/standards-guidelines/act/rules/e88epe/

It looks like the current rule wording fails this example:

```
<span>Rating:</span>
<img src='star.png' alt='5 Stars'>
<img src='star.png' alt=''>
<img src='star.png' alt=''>
<img src='star.png' alt=''>
<img src='star.png' alt=''>
```

This is voiced as 'Rating, 5 Stars, graphic'

The version which passes the rule is harder to understand:

```
<span>Rating:</span>
<img src='star.png' alt='Star'>
<img src='star.png' alt='Star'>
<img src='star.png' alt='Star'>
<img src='star.png' alt='Star'>
<img src='star.png' alt='Star'>
```

This is voiced as 'Rating: Star graphic, Star graphic, Star graphic, Star graphic, Star graphic'

**Comment by dan-tripp-siteimprove** [comment]
Hi Mark - FYI we discussed this in the meeting today, and the minutes will contain some thoughts on it.

**Comment by carlosapaduarte** [comment]
CG decision on [2025-06-26 meeting](https://www.w3.org/2025/06/26-act-r-minutes.html#item03): update the rule to pass scenarios where information is provided somewhere else within the page

**Comment by dd8** [comment]
Just a heads up - my original example is very close to Example 1 in this sufficient technique

https://www.w3.org/WAI/WCAG21/Techniques/general/G196

The test procedure in G196 should give some ideas on how to update the rule
